src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -676,25 +681,32 @@ namespace winrt::TerminalApp::implementation
     // - direction: The direction to move the pane in.
     // Return Value:
     // - <none>

```

Comment:
Can you fix some of these retval comments to explain the meaning of the new `bool` return?

Reply:
Ah, I missed that, will definitely add.